CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes for each version of this project will be documented in this
 - `IgxPivotGrid`
     - Added horizontal layout for row dimensions. Can be configured through the `pivotUI` `rowLayout` property.
     - Added `horizontalSummary` property for each IPivotDimension, enabling summary row when using horizontal layout.
+    - Added `rowSummariesPosition` property to the `pivotUI`, configuring horizontal summaries position.
```

Comment:
Does `rowSummariesPosition` position work outside of a horizontal scenario? Because the name doesn't really reflect that and we would have to explain in the API description.. which doesn't exists. Might want to consider a better name if possible.
Also if `horizontalSummary` is also new and bool if compatible could it just be `'none' | 'top' | 'bottom'` or does the position setting need to be specified globally for all dimensions?